rcp/org.testeditor.rcp4.tests/src/org/testeditor/rcp4/dialogs/NetworkConnectionSettingDialogTest.xtend

```diff
@@ -0,0 +1,164 @@
+/** 
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ */
+package org.testeditor.rcp4.dialogs
+
+import static org.mockito.Mockito.mock
+import static org.mockito.Mockito.when
+import static java.nio.charset.StandardCharsets.*
+import java.io.File
+import org.eclipse.core.net.proxy.IProxyData
+import org.eclipse.core.net.proxy.IProxyService
+import org.eclipse.core.runtime.preferences.IEclipsePreferences
+import org.eclipse.e4.core.contexts.IEclipseContext
+import org.eclipse.swt.widgets.Shell
+import org.junit.Test
+import org.testeditor.dsl.common.testing.AbstractTest
+import org.mockito.Mock
+import com.google.common.io.Files
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+
+/** 
+ * Modultest for NetworkConnectionSettingDialog.
+ */
+class NetworkConnectionSettingDialogTest extends AbstractTest {
+
+	@Rule public TemporaryFolder tempFolder = new TemporaryFolder();
+
+	@Mock Shell parentShell
+	@Mock IEclipsePreferences prefs
+	@Mock IEclipseContext context
+
+	@Test def void testWorkOfflineSetting() throws Exception {
+		// Given
+		when(prefs.getBoolean("workOffline", false)).thenReturn(true)
+		var NetworkConnectionSettingDialog dialog = new NetworkConnectionSettingDialog(parentShell, prefs, null)
```

Comment:
This line is repeated throughout all the tests. Can we move it into a `@Before` method?